Makefile

```diff
@@ -1,12 +1,20 @@
 
-all: dev clean test dists release
+all: dev clean test types dists release
 
 dev:
 	pip install -q ".[dev]"
 
 test:
 	nosetests --with-coverage --cover-package=followthemoney --cover-erase --cover-html --cover-html-dir=coverage-report
 
+types:
+	mypy --strict followthemoney/types \
+		followthemoney/schema.py \
+		followthemoney/property.py \
+		followthemoney/proxy.py \
+		followthemoney/graph.py \
+		followthemoney/model.py
```

Comment:
Is the plan here that we add more and more files to this list as they're converted to be typed?

Reply:
Yep. It establishes an explicit coverage as long as the whole package isn't compliant.